Automattic/jetpack
# Command Palette: Remove Github deployments feature check

# We need to wait until March 18th (GitHub deployments feature release date) before merging

Fixes:
- https://github.com/Automattic/dotcom-forge/issues/6074

## Proposed changes:

* Removed the GitHub deployments feature check from the Command Palette config.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Load the plugin using Jetpack beta or manually uploading
* Visit an Atomic and/or Simple site
* Open your browser's console, and type `commandPaletteConfig` 
* Inspect the object, you should see that there is no `isGHDeployAvailable` property.
